projects/packages/publicize/src/class-publicize-script-data.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Publicize_Script_Data.
+ *
+ * @package automattic/jetpack-publicize
+ */
+
+namespace Automattic\Jetpack\Publicize;
+
+use Automattic\Jetpack\Current_Plan;
+use Automattic\Jetpack\Publicize\Publicize_Utils as Utils;
+
+/**
+ * Publicize_Script_Data class.
+ */
+class Publicize_Script_Data {
```

Comment:
I am a bit confused here what the class name "Script data" means. Could we have a more descriptive doc if this describes the class the best?

Reply:
Actually the package we created in #38430 was initially named as `initial-state` and the class was also named as `Jetpack_Initial_State` but after some discussion with Garage team, we reached the conclusion to use the word `script data` which is what the initial state actually is. Also, WordPress uses the similar nomenclature for such data - `wp_add_inline_script` has calls the second argument as `$data` and likewise the other such functions.